bin/smack.py

```diff
@@ -222,18 +301,23 @@ def clang_frontend(args):
 
   smack_root = os.path.dirname(os.path.dirname(os.path.abspath(sys.argv[0])))
   smack_headers = os.path.join(smack_root, 'share', 'smack', 'include')
-  smack_lib = os.path.join(smack_root, 'share', 'smack', 'lib', 'smack.c')
+  smack_lib = os.path.join(smack_root, 'share', 'smack', 'lib')
   smack_bc = temporary_file('smack', '.bc', args)
+  smack_svcomp_bc = temporary_file('smack-svcomp', '.bc', args)
 
   compile_command = ['clang', '-c', '-emit-llvm', '-O0', '-g', '-gcolumn-info']
   compile_command += args.clang_options.split()
-  compile_command += ['-I' + smack_headers, '-include' + 'smack.h']
+  compile_command += ['-I' + smack_headers, '-include' + ('smack-svcomp.h' if args.svcomp else 'smack.h')]
```

Comment:
Please merge {`smack.h`, `smack-svcomp.h`} and {`smack.c`, `smack-svcomp.c`} and revert this code.